_config.yml

```diff
@@ -1,30 +1,30 @@
 # Setup
-title             : Bootstrap Blog
-description       : Official blog for the Bootstrap framework.
-url               : "https://blog.getbootstrap.com"
-future            : true
-permalink         : pretty
+title:              "Bootstrap Blog"
```

Comment:
The quotes are not necessary actually. Is there a reason why you're bringing them back..?

Reply:
Yeah, I'm going with the safe way http://www.yamllint.com/

Reply:
Okay. Fair enough :smiley:

Reply:
I just need a way to (semi) automate the validity of the file, that's all :)

Reply:
If formatting and comments were not an issue, automation could be carried out by a Rake task or a standalone Ruby script. :smiley: 
```ruby
require "safe_yaml/load"

config_file = "./_config.yml"
config = SafeYAML.load_file(config_file)
File.write(config_file, YAML.dump(config))
```